content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -179,6 +182,6 @@ Watchdog: Launching child 1
 
```

Comment:
Another step you may want to add, either to "Next Steps" or as an optional last step before "3. Run rippled" is to run the unit tests:

```
./rippled -u
```

If there are no test failures, then you can be fairly certain that your `rippled` executable compiled correctly.